vllm/model_executor/models/gemma2.py

```diff
@@ -461,3 +463,50 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                            if self.config.tie_word_embeddings else None),
         )
         loader.load_weights(weights)
+
+
+class Gemma2EmbeddingModel(nn.Module, SupportsPP):
```

Comment:
I'm preemptively moving them into the same file to be consistent with the upcoming BERT PR. (#9056)